text/events.md

```diff
@@ -0,0 +1,144 @@
+# Using Events
+
+`pallets/simple-event`
+[
+	![Try on playground](https://img.shields.io/badge/Playground-Try%20it!-brightgreen?logo=Parity%20Substrate)
+](https://playground-staging.substrate.dev/?deploy=recipes&files=%2Fhome%2Fsubstrate%2Fworkspace%2Fpallets%2Fsimple-event%2Fsrc%2Flib.rs)
+[
+	![View on GitHub](https://img.shields.io/badge/Github-View%20Code-brightgreen?logo=github)
+](https://github.com/substrate-developer-hub/recipes/tree/master/pallets/simple-event/src/lib.rs)
+
+`pallets/generic-event`
+[
+	![Try on playground](https://img.shields.io/badge/Playground-Try%20it!-brightgreen?logo=Parity%20Substrate)
+](https://playground-staging.substrate.dev/?deploy=recipes&files=%2Fhome%2Fsubstrate%2Fworkspace%2Fpallets%2Fgeneric-event%2Fsrc%2Flib.rs)
+[
+	![View on GitHub](https://img.shields.io/badge/Github-View%20Code-brightgreen?logo=github)
+](https://github.com/substrate-developer-hub/recipes/tree/master/pallets/generic-event/src/lib.rs)
+
+Having a [transaction](https://substrate.dev/docs/en/knowledgebase/getting-started/glossary#transaction) included in a
+block does not guarantee that the function executed successfully. To verify that
+functions have executed successfully, emit an
+[event](https://substrate.dev/docs/en/knowledgebase/getting-started/glossary#events) at the bottom of the function body.
+
+Events notify the off-chain world of successful state transitions.
+
+## Some Prerequisites
+
+When using events, we have to include the `Event` type in our configuration trait. Although the
```

Comment:
"Although the syntax is a bit complex, it is the same every time." 

suggest /inform what is complex about the syntax or instead say "The Syntax is constant in each config" .. otherwise this creates questions

Reply:
The following sentence tries to explain this a bit:

> If you are a skilled Rust programmer you will
recognize this as a series of [trait bounds](https://doc.rust-lang.org/book/ch10-02-traits.html). If
you don't recognize this feature of Rust yet, don't worry; it is the same every time, so you can
just copy it and move on.

But also, this PR is about the ToC structure, so if we do want to change this wording, I would prefer to do it in another PR.